Makefile

```
@@ -29,7 +29,7 @@ typecheck:
 	npm run typecheck
 
 optimize-svg:
-	$(NODE_BIN)/svgo --config svgo.config.js -f src/img
+	$(NODE_BIN)/svgo --config svgo.config.js -r -f src/img
```

one tip I've seen is to always use long flag names when things are checked in to source so they document themselves better
```suggestion
	$(NODE_BIN)/svgo --config svgo.config.js --recursive --folder src/img
```

---

Yeah, that's a good point 👍 Will update and keep that in mind for the future.